### DISCUSSION TITLE
Add a GPUParticles3D Gravity Curve

### DISCUSSION THREAD

**Discussion by DannyDevSe** [godotengine/godot-proposals#10419] [3D]
Would be cool to be able to increase the gravity over time on particles similar to the scale curve but maybe separate curves for each axis

**Comment by Calinou** [comment]
Out of curiosity, what kind of effect are you trying to achieve? Can you post videos/GIFs of the kind of particle effect you're trying to recreate in Godot?

**Reply by DannyDevSe** [reply]
Fountain effect https://youtu.be/C0yiE4Xh-OQ?si=oFz_P8ZpQhlbo8Bz can make it by splitting it up into 2 parts or have a particle collision at the top with bounce or attractor, but I think a gravity curve would be cool maybe there already is an option to make this effect easily?

**Reply by DannyDevSe** [reply]
Unity seams to have something called force over lifetime, maybe that's similar
